creme/geolocation/migrations/0006_v2_7__settingvalue_use_entity_icon.py

```diff
@@ -0,0 +1,22 @@
+from django.db import migrations
+
+
+def populate_settingkey(apps, schema_editor):
+    from creme.geolocation import setting_keys
+
+    SettingValue = apps.get_model('creme_core', 'SettingValue')
+
+    SettingValue.objects.update_or_create(
+        key_id=setting_keys.use_entity_icon_key.id,
+        defaults={'json_value': False}
+    )
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('geolocation', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RunPython(populate_settingkey),
+    ]
```

Comment:
Hum rework populate.py to use `SettingValue.objects.get_or_create()` (see other script like activities' one) before the early `return`, then this migration script will be useless.